django-stubs/db/models/fields/related.pyi

```diff
@@ -242,7 +242,7 @@ _Through = TypeVar("_Through", bound=Model)
 _To = TypeVar("_To", bound=Model)
 
 class ManyToManyField(RelatedField[Any, Any], Generic[_To, _Through]):
-    description: str
+    description: _StrOrPromise
```

Comment:
Maybe it is a good time to use `StrOrPromise` instead?

Reply:
Do you mean importing it from `django_stubs_ext`?

Reply:
Yes, this is non blocking. We can do this later.

Reply:
Perhaps we should do the whole file at once? To not introduce any ambiguity?

Reply:
Yes, I agree :)
Maybe even for the whole project.